blueprint-files/ember-cli-typescript/tsconfig.json

```diff
@@ -1,25 +1,67 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "allowJs": true,
+    "target": "ES2022",
+    "module": "ES2022",
     "moduleResolution": "node",
```

Comment:
Not actionable yet, but I suspect we're going to want to change this to `node12` as soon as that graduates from nightly, since Webpack 5 (and so by extension, `ember-auto-import` and Embroider) honor `exports` config.

Reply:
Yep. Hoping they ship that in 4.7 and we can flip it on. 🤞🏼